mission/config/subconfigs/vehicle_respawn_info.hpp

```diff
@@ -813,438 +840,538 @@ class vehicles {
+	/*
+	class vn_o_static_rsna75 {
+		tags[] = {"pavn","static"};
+	};
+	class vn_sa2 {
+		tags[] = {"pavn","static"};
+	};
+	*/
 };
 
-class spawn_point_types {
-	class light_transport {
-		name = "Light transport";
-		RESPAWN_SHORT;
-		class categories {
-			class m151 {
-				name = "M151A1";
-				icon = VEHICLE_ICON_CAR;
-				include[] = { { "m151a1","transport","army" } };
-				exclude[] = { { "armed" } };
-			};
 
-			class mule {
-				name = "M274 Mule";
-				icon = VEHICLE_ICON_CAR;
-				include[] = { { "m274","transport" } };
-				exclude[] = { { "airport" } };
-			};
+/*
+TEMPLATES FOR WHAT VEHICLES PEOPLE CAN GET FROM DIFFERENT SPAWN POINTS
+
+i.e. applying the tags above to possible spawner
+
+A NOTE ON NAMING CONVENTIONS -------------------------------------------------------------------
+
+All classes here use a naming convention following this pattern: locking_type_subtype_othersubtypes
+
+- locking --> who the vehicle is locked to, if any
+- whether the vehidle is ground (grnd), air (air), static weapon (statics) or a boat (boats)
+- subtypes are: firesupport, transport, patrol, utility etc.
+- further subtypes are things like: custom, light, heavy, medium, airport etc.
+
+please add additional subtypes can be added at the end, e.g. unlocked_utility_airport_light
+*/
+
+class parent_grnd_light_transport {
+	name = "Ground Transport (Light)";
+	class categories {
+		class m151 {
+			name = "M151A1";
+			icon = VEHICLE_ICON_CAR;
+			include[] = { { "m151a1","transport","army" } };
+			exclude[] = { { "armed" } };
+		};
+		class mule {
+			name = "M274 Mule";
+			icon = VEHICLE_ICON_CAR;
+			include[] = { { "m274","transport" } };
+			exclude[] = { { "airport" } };
 		};
 	};
+};
 
-	class transport_airport {
-		name = "Light airport transport";
-		RESPAWN_SHORT;
+class parent_grnd_patrol {
+	name = "Ground Transport (Patrol)";
+	class categories {
+		class m151 {
+			name = "M151A1";
+			icon = VEHICLE_ICON_CAR;
+			include[] = { { "m151a1","armed","army" } };
+			exclude[] = { { "transport" } };
+		};
+		class mule {
+			name = "M274 Mule";
+			icon = VEHICLE_ICON_CAR;
+			include[] = { { "m274", "armed" } };
+			exclude[] = { { "transport" } };
+		};
+	};
+};
 
-		class categories {
-			class mule {
-				name = "M274 Mule";
-				icon = VEHICLE_ICON_CAR;
-				include[] = { { "m274","airport" } };
-				exclude[] = { { "armed" } };
-			};
-			class truck {
-				name = "Truck";
-				icon = VEHICLE_ICON_TRUCK;
-				include[] = { { "truck", "airport" } };
-				exclude[] = { { "armed" } };
-			};
+class parent_grnd_transport_heavy {
+	name = "Ground Transport (Heavy)";
+	class categories {
+		class m113 {
+			name = "M113";
+			icon = VEHICLE_ICON_ARMOUR;
+			include[] = { { "m113", "transport" } };
+			exclude[] = { { "firesupport" } };
+		};
+		class trucks {
+			name = "Trucks";
+			icon = VEHICLE_ICON_TRUCK;
+			include[] = { { "truck", "transport" } };
+			exclude[] = { { "airport" } };
 		};
 	};
```

Comment:
not currently in use so no live configuration added, although a `parent_grnd_transport_airport` template has been added